packages/visx-annotation/src/components/EditableAnnotation.tsx

```diff
@@ -142,20 +148,22 @@ export default function EditableAnnotation({
           fill="transparent"
         />
       )}
-      <circle
-        cx={subjectX}
-        cy={subjectY}
-        transform={`translate(${subjectDrag.dx},${subjectDrag.dy})`}
-        onMouseMove={subjectDrag.dragMove}
-        onMouseUp={subjectDrag.dragEnd}
-        onMouseDown={subjectDrag.dragStart}
-        onTouchStart={subjectDrag.dragStart}
-        onTouchMove={subjectDrag.dragMove}
-        onTouchEnd={subjectDrag.dragEnd}
-        cursor={subjectDrag.isDragging ? 'grabbing' : 'grab'}
-        {...defaultDragHandleProps}
-        {...subjectDragHandleProps}
-      />
+      {canEditSubject && (
+        <circle
+          cx={subjectX}
+          cy={subjectY}
+          transform={`translate(${subjectDrag.dx},${subjectDrag.dy})`}
+          onMouseMove={subjectDrag.dragMove}
+          onMouseUp={subjectDrag.dragEnd}
+          onMouseDown={subjectDrag.dragStart}
+          onTouchStart={subjectDrag.dragStart}
+          onTouchMove={subjectDrag.dragMove}
+          onTouchEnd={subjectDrag.dragEnd}
+          cursor={subjectDrag.isDragging ? 'grabbing' : 'grab'}
```

Comment:
Would we want this cursor state to configurable? maybe through `subjectDragHandleProps`?

Reply:
Yep! `{...subjectDragHandleProps}` is spread below this line so it should override `cursor` (tho currently it doesn't have access to `subjectDrag` state, is that what you meant?)